build/stf-run-ci/tasks/main.yml

```
@@ -37,9 +37,15 @@
   tags:
     - deploy
 
+- name: Set default base dir if not provided
+  set_fact:
+    name: base_dir
+    value: "{{ playbook_dir }}"
```

This should be: 
 
```suggestion
    base_dir: "{{ playbook_dir }}"
```

You've set two facts here called `name` and `value` with the values `"base_dir"` and `playbook_dir`
base_dir remains unset and the functionality remains the same as before.

---

I'm just going to commit this change to get it tested. I am >90% certain it'll resolve the error seen in the ci, which appeared to still be using the relative path.

---

Haha, how embarrassing! Thanks for the patch.